File: Smodr/src/main/java/com/kevintcoughlin/smodr/viewholders/EpisodeViewHolder.java
package com.kevintcoughlin.smodr.viewholders;

import android.widget.TextView;

import androidx.recyclerview.widget.RecyclerView;

import com.kevintcoughlin.smodr.databinding.ItemListEpisodeLayoutBinding;

public final class EpisodeViewHolder extends RecyclerView.ViewHolder {
    protected TextView mTitle;
    protected TextView mMetadata;
    protected TextView mDescription;

    EpisodeViewHolder(final ItemListEpisodeLayoutBinding binding) {
        super(binding.getRoot());

        mTitle = binding.title;
        mMetadata = binding.metadata;
        mDescription = binding.description;
    }
}

File: Smodr/src/main/java/com/kevintcoughlin/smodr/viewholders/EpisodeViewHolder.kt
package com.kevintcoughlin.smodr.viewholders

import androidx.recyclerview.widget.RecyclerView
import com.kevintcoughlin.smodr.databinding.ItemListEpisodeLayoutBinding

class EpisodeViewHolder(
    binding: ItemListEpisodeLayoutBinding
) : RecyclerView.ViewHolder(binding.root) {
    val mTitle = binding.title
    val mDescription = binding.description
    val mMetadata = binding.metadata
}